test/browser_testing/features/pages/closing_disclosure.py

```diff
@@ -107,8 +114,40 @@ def hover_an_overlay(self):
                     bad_elements += 1
                 else:
                     classes = filter(lambda x: x,
-                                    explainer_element.get_attribute('class').split(' '))
+                                     explainer_element.get_attribute('class').split(' '))
                     if 'has-attention' not in classes:
                         bad_elements += 1
 
         return bad_elements
+
+    def click_page(self, page_num):
+        elements = self.driver.find_elements_by_class_name('form-explainer_page-link')
+        for element in elements:
+            if element.get_attribute('data-page') == page_num:
```

Comment:
This is totally optional, but I was wondering if we could use the link's text instead of its `data-page` attribute? We might change the implementation so the links no longer need a data attribute, but their text value shouldn't change.